package.json

```diff
@@ -121,10 +124,13 @@
 		"codejar": "^3.5.0",
 		"d3-scale": "^4.0.2",
 		"debounce-promise": "^3.1.2",
+		"dotenv": "^16.0.3",
 		"optimade": "^2.1.0",
 		"optimade-mpds-nlp": "^0.1.3",
 		"sirv-cli": "^1.0.12",
 		"svelte-file-dropzone": "^0.0.15",
-		"tinycolor2": "^1.4.2"
+		"tinycolor2": "^1.4.2",
+		"znv": "^0.3.2",
+		"zod": "^3.20.2"
```

Comment:
`dotenv`, `znv`, `zod` - they should be dev dependencies. They are required only in build time (or dev server?). Browser app (runtime) can't read environment variables or `.env` file.

Reply:
Yes, they should be dev deps.
Environment variables are injected in build time via https://github.com/pyoner/metis-gui/blob/dev-docker/package.json#L95
Ok, I'll fix it.